compiler/include/symbol.h

```diff
@@ -383,6 +393,12 @@ class FnSymbol : public Symbol {
   bool            isMethod()                                   const;
   bool            isPrimaryMethod()                            const;
   bool            isSecondaryMethod()                          const;
+  bool            isIterator()                                 const;
```

Comment:
Were you suggesting that I find other places that were calling `->hasFlag(FLAG_ITERATOR_FN)` and replacing them with this?